src/index.html

```diff
@@ -1,5 +1,8 @@
 <!doctype html>
-<html lang="en">
+<html
+  class="html"
+  lang="en"
+>
```

Comment:
HTML Formatting: There is no need to add a class to the `html` element. It's not common practice and it's not necessary for styling.